yarn-project/end-to-end/src/e2e_avm_simulator.test.ts

```diff
@@ -64,6 +64,40 @@ describe('e2e_avm_simulator', () => {
         expect(tx.status).toEqual(TxStatus.MINED);
       });
     });
+
+    describe('ACVM interoperability', () => {
+      it('Can execute ACVM function among AVM functions', async () => {
+        expect(await avmContract.methods.constant_field_acvm().simulate()).toEqual([123456n, 0n, 0n, 0n]);
+      });
+
+      it('Can call AVM function from ACVM', async () => {
+        expect(await avmContract.methods.call_avm_from_acvm().simulate()).toEqual([123456n, 0n, 0n, 0n]);
+      });
+
+      it('Can call ACVM function from AVM', async () => {
+        expect(await avmContract.methods.call_acvm_from_avm().simulate()).toEqual([123456n, 0n, 0n, 0n]);
+      });
+
+      // Cannot work because ACVM does not support pending nullifiers.
+      // it('AVM->ACVM nullifiers work (pending)', async () => {
+      //   await avmContract.methods.avm_to_acvm_nullifier().send().wait();
+      // });
+
```

Comment:
Should this EVER work? If not, should we remove the commented code? If it SHOULD work someday, maybe we link a ticket?

Reply:
I think it will never work. Removed.